### ISSUE TITLE
Vue selector prevents linting vue template

### ISSUE THREAD

**Issue by chubei-oppen** [SublimeLinter/SublimeLinter-eslint#176]
Fill your issue here.

with following vue file:
```
<template>
</template>

<script>
</script>

<style>
</style>
```
and configuration:
```
{
    "root": true,
    "env": {
        "browser": true,
        "es6": true
    },
    "extends": [
        "eslint:recommended",
        "google",
        "plugin:vue/recommended"
    ],
    "parserOptions": {
        "parser": "babel-eslint",
        "sourceType": "module",
        "allowImportExportEverywhere": true
    }
}
```

#### `SublimeLinter` console output

SublimeLinter: eslint: test.vue ['E:\\Workspace\\webar\\node_modules\\.bin\\eslint.cmd', '--format', 'compact', '--stdin', '--stdin-filename', '@'] 

#### `eslint` output from command line

E:\Workspace\webar\front-end\src\test.vue
  1:1  error  The template root requires exactly one element  vue/valid-template-root

✖ 1 problem (1 error, 0 warnings)

#### Additional info if needed (ST version, OS, configuration)

After removing line 43 of linter.py:

SublimeLinter: eslint: test.vue ['E:\\Workspace\\webar\\node_modules\\.bin\\eslint.cmd', '--format', 'compact', '--stdin', '--stdin-filename', '@'] 
SublimeLinter: eslint output:
E:\Workspace\webar\front-end\src\test.vue: line 1, col 1, Error - The template root requires exactly one element. (vue/valid-template-root)

1 problem

**Comment by chubei-oppen** [comment]
Actually there has been a pull request for this. #169

**Comment by kaste** [comment]
Fixed via #169
